frontend/packages/db-structure/src/parser/sql/postgresql/converter.ts

```diff
@@ -69,12 +69,14 @@ const constraintToRelationship = (
     throw new Error('Invalid foreign key constraint')
   }
 
-  const name = defaultRelationshipName(
-    primaryTableName,
-    primaryColumnName,
-    foreignTableName,
-    foreignColumnName,
-  )
+  const name =
+    constraint.conname ??
+    defaultRelationshipName(
+      primaryTableName,
+      primaryColumnName,
+      foreignTableName,
+      foreignColumnName,
+    )
```

Comment:
📝 `ALTER TABLE` uses conname and `CREATE TABLE` uses defaultRelationshipName, so we have now standardised them both.